### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by odaialazzeh**
- Update index.html and used the flexbox to arrange the containers to fit the desktop version.
- Update style.css and added the media queries to fit the desktop version.
- Add the images for the desktop version.
- Update README file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,68 +54,83 @@ <h1>Hey there. I’m Odai <span>I’m a software developer</span></h1>
...
+                        <div class="sub-container">
+                            <h3>Multi-Post Stories Gain+Glory</h3>
+                            <ul>
+                                <li>Ruby on rail</li>
+                                <li>css</li>
+                                <li>JavScript</li>
+                                <li>html</li>
+                            </ul>
+                            <a href="#">See Project</a>
+                        </div>
```
Comment (mckent05) [inline index.html:94]: - Good job so far @odaialazzeh 👏 👏 . I noticed that the project cards in  your work section (mobile version) are missaligned, please check and confirm what the issues could be 👍 👍.

How your project looks:
project card student

File: index.html [hunk 2]
```diff
@@ -111,11 +139,18 @@ <h3>Multi-Post Stories Gain+Glory</h3>
...
+                        <img id="img7" src="images/Mask Group.svg" alt="">
+                        <img id="img8" src="images/Frame-about.svg" alt="">
+                    </div>
                     <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                     <a href="#">Get My Resume</a>
-                    <img id="img8" src="images/Frame-about.svg" alt="">
                 </div>
+                <div class="main-container-about">
                     <div class="container-about">
                         <div class="a-img">
```
Comment (mckent05) [inline index.html:155]: - Good job done 👏 . But I noticed the technology cards in the about me section in the mobile version of your project are misaligned. Please try to fix this issue 🙏 🙏.

How your project looks currently:
about me student

File: index.html [hunk 3]
```diff
@@ -157,29 +192,55 @@ <h3>Languages</h3>
...
+                    </form>
+                </div>                    
+                <div class="social-media-icons">
+                    <a href="#"> <img src="images/twitter.svg" alt=""> </a>
+                    <a href="#"> <img src="images/Linkedin icon.svg" alt=""> </a>
+                    <a href="#"> <img src="images/medium.svg" alt=""> </a>
+                    <a href="#"> <img src="images/github.svg" alt=""> </a>
+                    <a href="#"> <img src="images/angellist.svg" alt=""> </a>
+                </div>
+                <div class="form-img"><img id="img-13" src="images/Frame-1.svg" alt="">                </div>
```
Comment (mckent05) [inline index.html:238]: - To improve your HTML semantics and your site's accessibility, I'd suggest you wrap each of your links in a `li` tag and wrap all `li` tags in a `ul` tag. Please fix this😄 😄

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @odaialazzeh , 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there


## Highlights
- Your project is looking nice and gradually getting complete 👏 👏 
- Your repo looks really professional 👍 👍 
- All linters are passing correctly ✔️ ✔️  

## Required Changes ♻️
- Please it is important you check for bugs in the mobile version of your project when changes are made to the desktop version. Please note this when working on future projects 👍 😄 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @odaialazzeh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

-   No linter errors. :heavy_check_mark: 
-   Professional README file. :heavy_check_mark: 
-   Descriptive PR Title. :heavy_check_mark: 
-   Meaningful commit messages. :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**